examples/src/test/kotlin/AggregatesBuilderTest.kt

```diff
@@ -951,6 +952,40 @@ class AggregatesBuilderTest {
         assertEquals("Back to the Future", results.first().title)
     }
 
+    /* NOTE: Test is not run by default. FTS requires the creation of a text index on the collection before running
+    (see note at top of file for additional setup requirements for FTS).
+     */
+    @Ignore
+    fun atlasSearchOperatorTest() = runBlocking {
+
+        // :snippet-start: atlas-search-pipeline
+        data class Results(val title: String, val year: Int, val genres: List<String>)
+
+        val searchStage = Aggregates.search(
+            SearchOperator.compound()
+                .filter(
+                    listOf(
+                        SearchOperator.`in`(SearchPath.fieldPath(Movie::genres.name), listOf("Comedy")),
+                        SearchOperator.phrase(SearchPath.fieldPath(Movie::fullplot.name), "new york"),
+                        SearchOperator.numberRange(SearchPath.fieldPath(Movie::year.name)).gtLt(1950, 2000),
+                        SearchOperator.wildcard(SearchPath.fieldPath(Movie::title.name), "Love *")
+                    )
+                )
+        )
+
+        val projectStage = Aggregates.project(
+            Projections.include(Movie::title.name, Movie::year.name, Movie::genres.name))
```

Comment:
should we exclude `_id` explicitly above? Feel free to ignore it if the pattern is well-established already.
Just wondering without `_id` suppression, it will show up in the result.

Reply:
Since this example deserializes into the defined `Results` data class, the Id isn't showing up in the results. For the other examples (kotlin sync, java, etc), projection is not the focus of this example, so I am indifferent to showing/excluding the `_id` field.